src/cbc_sdk/platform/alerts.py

```diff
@@ -420,44 +422,54 @@ def workflow_(self):
         """
         return self._workflow
 
-    def _update_workflow_status(self, state, remediation, comment):
+    def _update_workflow_status(self, status, closure_reason, determination, note):
         """
         Updates the workflow status of this alert.
 
         Args:
-            state (str): The state to set for this alert, either "OPEN" or "DISMISSED".
-            remediation (str): The remediation status to set for the alert.
-            comment (str): The comment to set for the alert.
+            status (str): The status to set for this alert, either "OPEN", "IN_PROGRESS", or "CLOSED".
+            closure_reason (str): the closure reason for this alert, either "TRUE_POSITIVE", "FALSE_POSITIVE", or "NONE"
+            determination (str): The determination status to set for the alert, either "NO_REASON", "RESOLVED",
+            "RESOLVED_BENIGN_KNOWN_GOOD", "DUPLICATE_CLEANUP", or "OTHER"
+            note (str): The comment to set for the alert.
         """
-        request = {"status": state}
-        if remediation:
-            request["closure_reason"] = remediation
-        if comment:
-            request["note"] = comment
-        url = self.urlobject.format(self._cb.credentials.org_key) + "/workflow"
-        resp = self._cb.post_object(url, request)
+        request = {}
+        if status:
+            request["status"] = status
+        if determination:
+            request["determination"] = determination
+        if closure_reason:
+            request["closure_reason"] = closure_reason
+        if note:
+            request["note"] = note
+
+        resp = self._cb.post_object(self.url, request)
         self._workflow = Workflow(self._cb, resp.json())
         self._last_refresh_time = time.time()
 
-    def dismiss(self, remediation=None, comment=None):
+    def dismiss(self, determination=None, closure_reason=None, note=None):
```

Comment:
will do

Reply:
It should also be changed to "close" instead of "dismiss".  It's been overhauled as a feature, the SDK should align since this is the chance to be a breaking change.